### ISSUE TITLE
'Hide on Launch' only when launching on windows startup

### ISSUE THREAD

**Issue by kiranjholla** [dragonwocky/obsidian-tray#57]
The Hide on Launch feature is a really useful one to ensure the window doesn't get in the way whenever the computer starts up. However, this feature also makes it such that the window is hidden even when obsidian is started manually. Hence, the user has no indication that the app has been launched.

Would it be possible to add an option to "Hide on Launch" only during startup and not when the app is launched manually?

**Comment by MahyarExplores** [comment]
I agree. just wanted to report the same thing. it would be great if it wasn't this way. its the biggest thing that needs changing right now!

possible solution:
just define a `bool IsStartup = true` and then when obsidian is hidden for the first time after startup, set the IsStartup to false. and each time you want to hide the window just check if IsStartup is true or false.
in addition to make it optional, you can make it so that if the user prefers, it wont check the bool variable.

I would do it immediately myself if i knew js
